router/src/components/FullPlayers.js

```diff
@@ -0,0 +1,79 @@
+import React, {Component} from 'react';
+import PlayerAPI from './api';
+import {Link} from 'react-router-dom';
+import PlayersAPI from './api';
+import '../style/FullPlayers.css'
+
+class FullPlayers extends Component{
+    constructor() {
+        super();
+        this.state = {
+            players: PlayerAPI.all()
+        }
+        this.delete = this.delete.bind(this);
+        this.change=this.change.bind(this);
+        this.addPlayerForm=this.addPlayerForm.bind(this);
+    }
+
+    delete(e) {
+        let newPlayers = this.state.players.filter(p => p.number.toString() !== e.target.id);
+        // console.log(newPlayers)
+        console.log(PlayerAPI.deleteHandle(e))
+        this.setState({
+            players: newPlayers
+        })
+    }
+    handleSubmit(){
+        let name=this.state.playerName
+        let number=parseInt(this.state.playerNumber,10)
```

Comment:
if you want to have ID for each player, try this -> 
npm install uuid
var uuid = require('uuid');

just like we did in firebase example

Reply:
@wizzardio I've made some changes (add id to each player & rewrite deleteHandle function) check it)